packages/libllvm/clang-lib-Driver-ToolChains-Linux.cpp.patch

```diff
@@ -28,7 +28,7 @@
  
    addPathIfExists(D, SysRoot + "/lib", Paths);
 -  addPathIfExists(D, SysRoot + "/usr/lib", Paths);
-+  bool nativeBuild = MultiarchTriple == llvm::sys::getDefaultTargetTriple();
++  bool nativeBuild = MultiarchTriple == getMultiarchTriple(D, llvm::Triple(llvm::sys::getDefaultTargetTriple()), SysRoot);
```

Comment:
Sigh, this line still isn't right, been struggling with getting it to work and submitted without making sure this version worked. I will update this pull once I'm sure I have it.